config/pre-install/v0.18.0/clusterrole.yaml

```diff
@@ -58,3 +58,16 @@ rules:
       - "delete"
       - "patch"
       - "watch"
+  - apiGroups:
```

Comment:
can we just add the groups and resources here to LOC 48 and 50?

Reply:
@nlopezgi you mean messaging api group should come before eventing api group? or something else?

Reply:
I think it should be possible to have a single `apiGroups` statement that includes the following resources:
```
      - "containersources"
      - "sinkbindings"	      
      - "channels"
      - "inmemorychannels"
      - "brokers"
      - "triggers"	
```
given that for all resources the same verbs are listed?
Just to reduce a bit of verbosity, but in all senses a nit, so totally your choice to leave as is or collapse.

Reply:
@nlopezgi not sure if that would work. These resources are from different apiGroups (eventing, messaging, etc.).
Let's leave as is and we can have a look later?

Reply:
thanks! yeah, you are probably right. I think with https://github.com/knative/eventing/pull/3943 my concern is addressed